regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -93,6 +93,11 @@ var PreambleView = ChildView.extend({
     this.mode = 'write';
     $parent = $parent.clone();
     $parent.find('.activate-write').remove();
+
+    if (section.indexOf('cfr') > -1) {
+      label = '§ ' + label;
+    }
```

Comment:
Is this needed? One of the major selling points of the `human_label` function in Python is that these types of inspection in JS land wouldn't be required